tests/acceptance/blockchain.controller.acceptance.tests.ts

```diff
@@ -0,0 +1,120 @@
+import { PocketGatewayApplication } from '../..'
+import { Client, expect } from '@loopback/testlab'
+import { setupApplication } from './test-helper'
+import { gatewayTestDB } from '../fixtures/test.datasource'
+import { BlockchainsRepository } from '../../src/repositories/blockchains.repository'
+import { Blockchains } from '../../src/models/blockchains.model'
+
+describe('Blockchain (acceptance)', () => {
+  let app: PocketGatewayApplication
+  let client: Client
+
+  before('setupApplication', async () => {
+    ;({ app, client } = await setupApplication())
+  })
+
+  after(async () => {
+    await app.stop()
+  })
+
+  const cleanDB = async () => {
+    await new BlockchainsRepository(gatewayTestDB).deleteAll()
+  }
+
+  afterEach(cleanDB)
+
+  it('retrieves list of available blockchains', async () => {
+    const expected = Object.assign(
+      {},
+      {
+        _id: '0024',
+        ticker: 'POA',
+        networkID: '42',
+        network: 'POA-42',
+        description: 'Kovan',
+        index: 6,
+        blockchain: 'poa-kovan',
+        active: true,
+        enforceResult: 'JSON',
+        nodeCount: 23,
+        hash: '1234',
+      }
+    )
+
+    await new BlockchainsRepository(gatewayTestDB).create(expected)
+
+    const res = await client.get('/blockchains').expect(200)
+
+    expect(res.body).to.be.Array()
+    expect(res.body).to.have.length(1)
+    expect(res.body[0]).to.containEql(expected)
+  })
+
+  it('retrieves the total count of blockchains', async () => {
+    await generateBlockchains(5)
```

Comment:
This method has side-effects, no? If so, I think we should at least return a boolean that the operation was successful and if not fail the test (because right now it feels like if it fails, it will just throw).